services/autoscaling/tests/unit/test_modules_rabbitmq.py

```diff
@@ -110,11 +110,11 @@ async def test_post_message(
     mocked_redis_server: None,
     initialized_app: FastAPI,
     rabbit_message: RabbitMessageBase,
-    rabbitmq_client: Callable[[str], RabbitMQClient],
```

Comment:
I would have separated this change of fixture name on its own PR

Reply:
Good point!